assets/src/admin/media-sharing/versionIcon.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
```

Comment:
If you want to import a package, make sure you add it to the package.json as a dependency.